src/Sylius/Behat/spec/Context/Ui/CartContextSpec.php

```diff
@@ -129,4 +134,37 @@ function it_ensures_there_is_no_discount_info_on_the_page(CartSummaryPage $cartS
 
         $this->thereShouldBeNoDiscount();
     }
+
+    function it_checks_if_cart_item_has_correct_discount_price(CartSummaryPage $cartSummaryPage, ProductInterface $product)
+    {
+        $cartSummaryPage->open()->shouldBeCalled();
+        $product->getName()->willReturn('Test product');
+
+        $cartSummaryPage->getItemDiscountPrice('Test product')->willReturn('€30.00');
+        $cartSummaryPage->getItemRegularPrice('Test product')->willReturn('€50.00');
+
+        $this->itsPriceShouldBeDecreasedBy($product, 2000);
```

Comment:
There is no spec for scenario when the price is not decreased by the given amount.